packages/frontend-shared/src/gql-components/topnav/UnsupportedBrowserTooltip.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div
+    ref="wrapper"
+    @mouseenter="showTooltip()"
+    @mouseleave="hideTooltip()"
+    @click="emit('click')"
+  >
+    <slot />
+    <transition
+      :name="`unsupported-browser-enter-to-${placement}`"
+      :enter-from-class="`transition-all transform ${placement === 'top' ? 'origin-bottom' : 'origin-top'} scale-y-0`"
+      :enter-to-class="`transition-all transform ${placement === 'top' ? 'origin-bottom' : 'origin-top'} scale-y-100`"
+    >
+      <div
+        v-if="scaleUp"
+        class="rounded flex h-fit
+                  bg-gray-900
+                  text-center py-16px px-24px
+                  text-gray-300 text-size-14px leading-20px
+                  absolute content
+                  need-content before:border-solid before:border-transparent before:border-width-8px
```

Comment:
Cool trick: You can group the `before:class-1 before:class2` by wrapping in parenthesis e.g. `before:(class-1 class-2)`. Makes for cleaner templates.